test/Mono.Linker.Tests/TestCases/TestSuites.cs

```diff
@@ -147,6 +147,12 @@ public void CommandLineTests (TestCase testCase)
 		[TestCaseSource (typeof (TestDatabase), nameof (TestDatabase.CodegenAnnotationTests))]
 		public void CodegenAnnotationTests (TestCase testCase)
 		{
+			if (Windows)
```

Comment:
Sorry, I should have been more clear.  When I said `Windows`, I meant "Replace me with the verbose way .NET makes you check if you are on windows"

```
SystemEnvironment.OSVersion.Platform == PlatformID.Win32
```

Reply:
I thought you had some variables inherited from test infrastructure or something. That makes more sense. Thanks